CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ All notable changes to this project will be documented in this file. This projec
     -   Allows maximum item-nesting of two levels
 -   Changes to MasterLayout
     -   The default values for content-spacing and header-height have changed slightly
+-   Removed graphql-tag as it gets re-exported by @apollo/client
```

Comment:
I don't think this change is incompatible? (or even worth mentioning?)

Reply:
You're probably right. I wanted to indicate that you should stop using it inside your application, but that's another thing...

Reply:
@nsams I've removed it,